tool/gravity/cli/operation.go

```diff
@@ -82,13 +82,8 @@ type SetPhaseParams struct {
 
 // resumeOperation resumes the operation specified with params
 func resumeOperation(localEnv *localenv.LocalEnvironment, environ LocalEnvironmentFactory, params PhaseParams) error {
-	err := executePhase(localEnv, environ, PhaseParams{
-		PhaseID:          fsm.RootPhase,
-		Force:            params.Force,
-		Timeout:          params.Timeout,
-		SkipVersionCheck: params.SkipVersionCheck,
-		OperationID:      params.OperationID,
-	})
+	params.PhaseID = fsm.RootPhase
```

Comment:
This fixes the issue with `gravity plan resume` introduced [here](https://github.com/gravitational/gravity/pull/1935) not relaying the `block` flag to the `executePhase` which decides how to execute - blocking or from service.